### DISCUSSION TITLE
Update download button as "Downloaded"

### DISCUSSION THREAD

**Discussion by Koronos** [civitai/civitai#140] [Ideas]
I have some problems identifing the models that I have already downloaded. I could suggest mark the models already downloaded as it?. Could be in the download button or a simbol.

This could help to reduce duplicate downloads. But without blocking a new download.

Thanks for your hard work is a really awesome project.

**Comment by aimerib** [comment]
I was literally about to ask for a `/me/downloads` api endpoint to be added so I could write my own chrome/firefox extension for this feature. Didn't realize this was a feature request already. Please Civitai, some of us are serious hoarders, and knowing what we've downloaded already without having to cross-reference the downloads page is a huge plus. Even when I order by newest, sometimes I've already downloaded something from early access, and upon release it floats back to the top of newest.

You have the data, you give us the data. Can we make it more convenient and meaningful to use the data?
